packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -2675,3 +2707,26 @@ typedef SelectableRegionContextMenuBuilder = Widget Function(
   BuildContext context,
   SelectableRegionState selectableRegionState,
 );
+
+/// A controller for a [SelectionArea] or [SelectableRegion].
+///
+/// Utilize this controller to programatically clear the content of a
+/// [SelectionArea] or [SelectableRegion], or select all the contents under it.
+class SelectionController with ChangeNotifier {
```

Comment:
Is this still the plan?

Reply:
After talking with @chunhtai, I think for now it might be less intrusive to expose `SelectableRegionState` and make its method `_clearSelection` public. This way we don't tie ourselves to any specific method in this PR.